app/themes/dashboard/ui/index.html

```diff
@@ -19,8 +19,8 @@
     <!-- endbuild -->
 
   </head>
-  <body ng-app="app" class="skin-black">
-
+  <body ng-app="app" class="skin-black sidebar-mini" ng-controller="SidebarController" sidebar-directive="sidebarState">
+    <toast></toast>
```

Comment:
Got it. I was having trouble working with the sidebar-open and sidebar-collapsed css classes in root.html.